test/LibraryManager.IntegrationTest/VisualStudioLibmanHostTests.cs

```diff
@@ -67,6 +67,14 @@ public SolutionService Solution
             }
         }
 
+        public string SolutionRootPath
+        {
+            get
+            {
+                return _solutionRootPath;
```

Comment:
>_solutionRootPath [](start = 23, length = 17)

Can it just be auto-property instead of having a backing field?

    public string SolutionRootPath { get; private set; }

Reply:
Yes. It can be. I can change it.

---
In reply to: [194547773](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194547773) [](ancestors = 194547773)